.github/workflows/build.yml

```diff
@@ -104,11 +115,11 @@ jobs:
         if [ "$RUNNER_OS" == "Windows" ]; then
           choco install innosetup
         fi
-        pip install poetry virtualenv
+        pip3 install poetry virtualenv
```

Comment:
I think if you do this, you're gonna have a bad time on Windows (iirc). 

Could be wrong, but getting these consistent has been a recurring issue.

Reply:
When I tested it on my fork (https://github.com/BelKed/activitywatch/actions/runs/2186226548), it ran well...
I have now added an `if` command for Windows in https://github.com/ActivityWatch/activitywatch/pull/733/commits/ad0c22a702365e29d0a3d467e636f9bebbc046dc, so I hope there will be no problems :)

Reply:
Ah, but if it really ran well then I guess it should be all good. I've just had one too many back-and-forth with these to always be a bit suspicious...

Reply:
Here you go: https://github.com/ActivityWatch/activitywatch/pull/733/commits/f69a89235c9a0830ef7ab5aec275e86e98e305cf